README.md

```diff
@@ -15,14 +15,14 @@
 
 ## Getting Started
 
-**A week 2 project of mobile version and desktop version of Figma template 3 for portfolio project.
+**A week 4 project of mobile version and desktop version of Figma template 3 for portfolio project.
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-HTML & CSS
+HTML, CSS & JS
```

Comment:
- [ ] Great job so far 👏. Your README looks good but **it would be better if you add a screenshot of your page**. I have seen your design and it looks great 💯 so **adding a screenshot of it will give the visitors a quick preview** of what your portfolio looks like therefore be more interested in your work 😉